src/content/docs/start-here/kafka-event.md

```diff
@@ -0,0 +1,470 @@
+
+The attributes of CloudEvent and the usage is described [here.](https://github.com/cloudevents/spec/blob/main/cloudevents/spec.md#required-attributes)
+
+For Kessel Inventory schema, look [here.](https://github.com/project-kessel/inventory-api/blob/main/data/kafka-event-schema.json)
+
+### Attributes 
+#### specversion (String)
+
+##### Cloud Event Intent
+The version of the CloudEvents specification which the event uses. This enables the interpretation of the context. Compliant event producers MUST use a value of 1.0 when referring to this version of the specification.
+
+##### Kessel Inventory Intent
+Same as above.
+
+#### type (String)
+##### Cloud Event Intent
+This attribute contains a value describing the type of event related to the originating occurrence. Often this attribute is used for routing, observability, policy enforcement, etc. The format of this is producer defined and might include information such as the version of the type - see Versioning of CloudEvents in the Primer for more information. Constraints:
+- MUST be a non-empty string 
+- SHOULD be prefixed with a reverse-DNS name.
+    - The prefixed domain dictates the organization which defines the semantics of this event type. <br> Examples:
+    ```
+    com.github.pull_request.opened 
+    com.example.object.deleted.v2
+    ```
+
+##### Kessel Inventory Intent
+We use a string comprised of __`redhat.inventory.resources.resource_type.operation`__ 
+```
+- redhat.inventory.resources.k8s-cluster.created 
+- redhat.inventory.resources.k8s-cluster.updated 
+- redhat.inventory.resources.k8s-cluster.deleted 
+```
+or <br>__`redhat.inventory.resource_relationships.relationship_type.operation`___ 
+```
+- redhat.inventory.resources-relationship.k8s-policy_ispropagatedto_k8s-cluster.created 
+- redhat.inventory.resources-relationship.k8s-policy_ispropagatedto_k8s-cluster.updated 
+- redhat.inventory.resources-relationship.k8s-policy_ispropagatedto_k8s-cluster.deleted 
+```
+The only valid values for operation are created, updated or deleted. This allows a consumer to filter on method and/or resource type.
+
+
+
+#### Source (URI)
+
+##### Cloud Event Intent
+Identifies the context in which an event happened. Often this will include information such as the type of the event source, the organization publishing the event or the process that produced the event. The exact syntax and semantics behind the data encoded in the URI is defined by the event producer. <br> <br>Producers MUST ensure that source + id is unique for each distinct event. <br> An application MAY assign a unique source to each distinct producer, which makes it easy to produce unique IDs since no other producer will have the same source. The application MAY use UUIDs, URNs, DNS authorities or an application-specific scheme to create unique source identifiers.
```

Comment:
```suggestion
Identifies the context in which an event happened. Often this will include information such as the type of the event source, the organization publishing the event or the process that produced the event. The exact syntax and semantics behind the data encoded in the URI is defined by the event producer. 

Producers MUST ensure that source + id is unique for each distinct event. 

An application MAY assign a unique source to each distinct producer, which makes it easy to produce unique IDs since no other producer will have the same source. The application MAY use UUIDs, URNs, DNS authorities or an application-specific scheme to create unique source identifiers.
```